cmd/cmd.go

```diff
@@ -209,8 +209,10 @@ func Run() error {
 	// bots enable
 	botsEnableCmd := botsCmd.Command("enable", "Enable a bot")
 	botsEnableID := botsEnableCmd.Arg("id", "ID of the bot").Required().String()
+	botsEnableCafe := botsEnableCmd.Flag("cafe-api", "Whether to serve bot on the Cafe API (public)").Short('c').Bool()
```

Comment:
the -c or --cafe-api flag during bot enable step will make it available over the cafe-api